.github/workflows/push.yml

```diff
@@ -0,0 +1,34 @@
+name: Check build
+
+on:
+  push:
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  builddebug:
+    name: Debug
+    uses: actions/checkout@v3
```

Comment:
github shows error here. Probably because it should be inside the `steps` keyword. See [here](https://github.com/klepsydra-technologies/kpsr-core/blob/main/.github/workflows/push.yml#L31)